.github/scripts/rebase-finish.sh

```diff
@@ -27,6 +34,9 @@ git push --set-upstream origin $ORIGIN_BRANCH-$TAG
 
 # Overwrite origin/$REBASE_BRANCH.
 git push --force-with-lease --set-upstream origin $ORIGIN_BRANCH
+
+# Clean up.
 git push origin -d $REBASE_BRANCH
+git push origin -d $BACKUP_BRANCH
```

Comment:
Why is backup branch getting deleted?

Reply:
How it looks from the workflow it might be renamed to unique name later. right?

Reply:
I don't think so. If the name would be unique, how'd you persist the information what was the name of the last backup branch accross the multiple workflow runs?

Reply:
why two backup branches are needed?

Reply:
We can stop creating `woarm64-*` branches if not usefull. Not a subject to this PR though.

Reply:
> $BACKUP\_BRANCH and woarm64-date-hash are different branches?

If the condition added above do not fail, they are pointing to the same commit. The `$BACKUP_BRANCH`, effectively resolving to `rebase-usptream-backup`, was added because one workflow run cannot determine what particular `woarm64-*` backup branch the previous run would create.